src/atc/functions.py

```diff
@@ -27,6 +28,7 @@ def init_dbutils():
     return dbutils
 
 
+@deprecated(details="Use DeltaHandle.drop_and_delete")
```

Comment:
This function is good if you dont know the exact path to the files on which the table is created. 
The function could might still be usefull for some.

Reply:
only deprecated.